.github/workflows/tests.yml

```diff
@@ -5,15 +5,15 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        go-version: [1.18.x, 1.19.x]
+        go-version: [1.16.x, 1.18.x, 1.19.x, stable]
```

Comment:
See https://github.com/google/starlark-go#stability: 
> We aim to support the most recent four (go1.x) releases of the Go toolchain. For example, if the latest release is go1.20, we support it along with go1.19, go1.18, and go1.17, but not go1.16.

Currently, the [latest release of Go is 1.21](https://go.dev/doc/devel/release#go1.21.0), so the oldest version we support is go1.18.

If you wouldn't mind changing this to [1.18.x, 1.19.x, 1.20.x, 1.21.x], that would be much appreciated.

Reply:
Changed. Also, modified `go.mod` file.